srcpkgs/lapce/patches/desktop-icon.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/extra/linux/dev.lapce.lapce.desktop b/extra/linux/dev.lapce.lapce.desktop
+index 83f9c5cc..8d81cd87 100644
+--- a/extra/linux/dev.lapce.lapce.desktop
++++ b/extra/linux/dev.lapce.lapce.desktop
+@@ -6,6 +6,6 @@ Name=Lapce
+ Comment=Lightning-fast and powerful code editor written in Rust
+ Categories=Development;IDE;
+ 
+-Icon=dev.lapce.lapce
++Icon=/usr/share/icons/hicolor/scalable/apps/dev.lapce.lapce.svg
```

Comment:
why is the original Icon not correct? Shouldn't it figure out the path?

Reply:
I haven't tested if it can figure the path, but giving the correct path will certainly be faster. It cannot, however, find the extension, at least in xfce.